apis/v1alpha1/gateway_types.go

```diff
@@ -105,6 +105,7 @@ type GatewaySpec struct {
 	// Support: Core
 	//
 	// +optional
+	// +kubebuilder:validation:MaxItems=16
```

Comment:
// +kubebuilder:validation:MinItems=0 and omitempty like other optional lists in this PR.

Reply:
Added omitempty, thanks!